Api/Settings.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace Drip\Connect\Api;
+use Drip\Connect\Api\SettingsInterface;
+
+class Settings implements SettingsInterface
+{
+		/** @var \Drip\Connect\Model\ConfigurationFactory */
```

Comment:
This whole file has somewhat goofed up indentation. Could this be fixed? We really need to get a linter on this project.